### DISCUSSION TITLE
请求新增接口：查询账号下所有 Key 的使用量

### DISCUSSION THREAD

**Discussion by DrayChou** [BurnHair/docs#181] [功能建议]
尊敬的团队成员，

我希望能在项目中新增一个功能接口，具体要求如下：

功能描述：

接口名称：查询账号下所有 Key 的使用量
功能目的：该接口将允许用户查询其账号下所有 API Key 的使用情况，以便于进行资源管理和使用监控。
需求细节：

输入参数：
账号标识（如用户ID或账号名）
输出结果：
每个 Key 的使用量统计（如调用次数、时间范围等）
可能的错误信息（如无效账号、权限不足等）
预期效果：

提高用户对 API Key 使用情况的透明度
帮助用户更好地管理和优化资源使用
请大家对此功能的可行性和实现方案进行讨论，期待你们的反馈和建议。

谢谢！

**Comment by BennyThink** [comment]
已经有了，请[参考文档](https://github.com/BurnHair/docs#11-%E8%8E%B7%E5%8F%96%E7%94%A8%E6%88%B7%E7%BB%9F%E8%AE%A1%E4%BF%A1%E6%81%AF)
